src/samples/audio/smart_amp_test.c

```diff
@@ -504,7 +504,7 @@ static int smart_amp_verify_params(struct comp_dev *dev,
 
 	ret = comp_verify_params(dev, BUFF_PARAMS_CHANNELS, params);
 	if (ret < 0) {
-		comp_err(dev, "volume_verify_params() error: comp_verify_params() failed.");
+		comp_err(dev, "smart_amp_verify_params() error: comp_verify_params() failed.");
```

Comment:
can you please reorder the patches in the PR to do: typos first, pre-req fixes next, new API definitions next followed by the IPC4 support for the smartamp module? spare a thought for CI and git bisect please.

Reply:
sure, thanks